doc/configuration.md

```diff
@@ -151,3 +155,95 @@ When calling the `</foo>` endpoint, both the team and location tags will be adde
 If `on`, and if `httpd` is making the trace sampling decision (i.e. if `httpd` is the first service in the trace), then delegate the sampling decision to the upstream service. nginx will make a provisional sampling decision, and convey it along with the intention to delegate to the upstream. The upstream service might then make its own sampling decision and convey that decision back in the response. If the upstream does so, then nginx will use the upstream's sampling decision instead of the provisional decision.
 
 Sampling delegation exists to allow `httpd` to act as a reverse proxy for multiple different services, where the trace sampling decision can be better made within the service than it can within `httpd`.
+
+# Configuring Real User Monitoring
+
+> [!IMPORTANT]
+> 🚧 RUM SDK Injection is currently in Private Beta! 🚧
+>
+> Interested in early access? [Apply to join the beta program today!](https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/)
+
+## `DatadogRum` directive
+   - **Description**: Enable or disable the RUM Injection
+   - **Syntax:** DatadogRum *On\|Off*
+   - **Default:** On
```

Comment:
```suggestion
   - **Default:** Off
```

The code actually behaves like this, right?

Reply:
Indeed